src/main/java/jaryl/duke/Output.java

```diff
@@ -0,0 +1,207 @@
+package jaryl.duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Output handles I/O, interactions with user
+ */
+public class Output {
+    /**
+     * Prints welcome message
+     */
+    public String printWelcomeMsg() {
+        return "Hello! I'm Duke, your friendly chatbot. What can I do for you today? Type help for more information.";
+    }
+
+    /**
+     * Prints message upon successfully adding task
+     * @param task      the task to be added
+     * @param numTasks  total number of existing tasks
+     */
+    public String printAddedTask(Task task, int numTasks) {
+        return "Got it. I've added this task:\n\t" + task + "\nNow you have " + numTasks + " tasks in the list.";
+    }
+
+    /**
+     * Handles help command
+     */
+    public String sendHelp() {
+        return ("Duke has sent help! Here are a list of commands you can use:\n1. list\n2. done\n" +
+                "3. find\n4. todo\n" + "5. deadline\n6. event\n" + "7. delete\n8. help\n9. exit");
+    }
+
+    /**
+     * Prints message upon successfully marking a task done
+     * @param task  the task to be marked done
+     */
+    public String printDoneMsg(Task task) {
+        if(task.getStatusIcon().equals("\u2713"))
```

Comment:
Should the if statement be wrapped in curly brackets even when they are single statement conditionals?

Reply:
Thanks Nicholas! I totally agree with you. This is part of my bad coding habit. Will revise.